articles/azure-developer-cli/faq.yml

```diff
@@ -56,6 +56,10 @@ sections:
       - question: How is the azure.yaml file used?
         answer: |
           The azure.yaml file describes the apps and types of Azure resources that are included in the template.
+        
+      - question: Can I use Azure Free Subscription?
+        answer: |
+          Yes. You can use Azure Free Subscription but you will have limitation where one Azure location can only be used for one template. If you see deployment error shown as "InvalidTemplateDeployment: The template deployment '<env_name>' is not valid according to the validation procedure. The tracking id is '<id>'. See inner errors for details", this means you have already used the selected Azure location. You can select another Azure location to fix the error.
```

Comment:
How about: 

"Yes. You can use Azure Free Subscription but each Azure location can only have one deployment. If you have already used the selected Azure location, you will see the deployment error: "InvalidTemplateDeployment: The template deployment '<env_name>' is not valid according to the validation procedure. The tracking id is '<id>'. See inner errors for details". You can select a different Azure location to fix the issue."